SparseGrids/TasmanianSparseGrid.cpp

```diff
@@ -220,6 +221,17 @@ void TasmanianSparseGrid::makeWaveletGrid(int dimensions, int outputs, int depth
         std::copy(level_limits, level_limits + dimensions, llimits);
     }
 }
+void TasmanianSparseGrid::makeFourierGrid(int dimensions, int outputs, int depth, TypeDepth type, const int* anisotropic_weights, const int* level_limits){
+    clear();
+    fourier = new GridFourier();
+    fourier->makeGrid(dimensions, outputs, depth, type, anisotropic_weights, level_limits);
+    base = fourier;
+    if(level_limits != 0) {
```

Comment:
* The code must be formatter to be consistent with the rest (as well as self-consistent)
    * sometimes you leave a space between `) {`, do not
    * one space after `if (`
    * spaces around `=`, e.g., `i = 1`, there are exceptions here but only when trying to save space and make the code compact
    * there are places with multiple empty lines between functions, use one or zero lines
    * remove all trailing spaces and empty lines

Reply:
I believe I have taken care of items 1, 2, 4, and 5.  

For item 3, I did a reg exp search of (at least most of) the documents where I've made changes. The only instances I saw were in `for` loops. Was there a place in particular where I needed to fix the spaces around `=`?

The regexp I used was `[^( |=|(for\(int (a-z)))]=[^( |=)]`

Reply:
The spaces around `=` go usually outside of the loop, I have to go back to find the spot where I saw it. Look over for code such as
```
base=fourier;
```

Reply:
I think I found it in `tasgridTestFunctions.cpp`.